src/mumble/AudioOutput.cpp

```diff
@@ -496,19 +496,15 @@ bool AudioOutput::mix(void *outbuff, unsigned int nsamp) {
 
 			if (recorder) {
 				AudioOutputSpeech *aos = qobject_cast<AudioOutputSpeech *>(aop);
+				Q_ASSERT(aos != nullptr);
```

Comment:
Also, since this is a qobject_cast, aos being null is not a bug. We should just remove the assert.